jdaviz/components/data_menu_remove.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-menu
+    absolute
+    offset-y
+    left
+  >
+    <template v-slot:activator="{ on, attrs }">
+      <j-tooltip
+        :span_style="'display: inline-block; float: right; ' + (delete_enabled ? '' : 'cursor: default;')"
+        :tooltipcontent="delete_tooltip"
+      >
+        <v-btn
+            icon
+            v-bind="attrs"
+            v-on="on"
+            :disabled="!delete_enabled"
+            >
+            <v-icon class="invert-if-dark">mdi-delete</v-icon>
+        </v-btn>
+      </j-tooltip>
+    </template>
+    <v-list dense style="width: 200px">
+      <v-list-item>
+        <v-list-item-content>
+          <j-tooltip tooltipcontent="Remove from this viewer only">
+            <span
+              style="cursor: pointer; width: 100%"
+              @click="() => {$emit('remove-from-viewer')}"
+            >
+              Remove from viewer
```

Comment:
Very nit-picky on the phrasing, but it bothers me a little bit that for data, this actually removes the data from the viewer, but with a subset, it just makes it invisible. Could we have conditional text here to reflect what is actually happening in the subset case? I know that's extra complicated since you could select both a subset and a data layer, this isn't a deal breaker for me.

Reply:
yeah, agreed.  We could handle this with the text and/or tooltip changing based on the selection or we could disable it entirely if a subset is selected.  @Jenneh - thoughts?

Reply:
@rosteen what do you think of updating the tooltip for subsets to say "hide from viewer" rather than "remove selected data"? would that be enough to clarify?

Reply:
how is this?


https://github.com/user-attachments/assets/f2f2d53f-e081-4907-a866-7426d179fe75